src/index.js

```diff
@@ -29,7 +29,7 @@ function App(props) {
     !isAuthenticating && (
       <UserContext.Provider value={{ user, setUser }}>
         <BrowserRouter>
-          <Routing appProps={{ authenticated, setAuthenticated, user }} />
+          <Routing appProps={{ authenticated, setAuthenticated }} />
```

Comment:
we are sure this is the last page that needs the prop removed?

if so, this change is ok